### PR TITLE
Test device numbers

### PR DESCRIPTION
**Pull request by fntlnz**
Wrote a test to be sure that calculated major and minor numbers for devices are working as expected when dealing with devices during copy.

Signed-off-by: Lorenzo Fontana <lo@linux.com>

### CODE REVIEW

File: fileutils.go [hunk 1]
```diff
@@ -139,19 +139,16 @@ func CopyDirectory(source string, dest string) error {
 			return nil
 		}
 
-		// Copy the file.
-		if err := CopyFile(path, filepath.Join(dest, relPath)); err != nil {
-			return err
-		}
-
-		return nil
+		return CopyFile(path, filepath.Join(dest, relPath))
```
Comment (fntlnz) [inline fileutils.go:142]: Here I just noted that checking for the error was a bit redundant, if there's was a reason for that just let me know and I'll send without this

**Comment by fntlnz** [comment]
@mrunalp PTAL

File: fileutils.go (hunk 1 above)
Comment (mrunalp) [reply fileutils.go:142]: :+1:

**Comment by mrunalp** [comment]
LGTM. Thanks!

**Pull request merged**